virttest/utils_net.py

```diff
@@ -3105,7 +3105,7 @@ def __arping(ip, macs, dev, timeout):
         arping_cmd = "%s -f -c3 -w%d -I %s %s" % (arping_bin, int(timeout),
                                                   dev, ip)
         try:
-            o = process.system_output(arping_cmd, shell=True)
```

Comment:
For arping output, we may retain verbose ouput to debug purpose

Reply:
I would think if it is for the debug purpose, one should enable it when running the case that needs to be debugged. 
Not all cases.